pkgs/tools/networking/bgpdump/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchzip, autoconf, zlib, bzip2 }:
+
+stdenv.mkDerivation rec {
+  name = "bgpdump-94a0e724b335";
```

Comment:
The version should be in date format, so `2017-09-29`. In nix, the version number has to start with a digit, and commit hashes can start with letters, so using a date that points to when the commit happened is preferred to using the commit hash itself for 'unstable' versions.